Update getting-started.md to use .NET 5

macel94 opened the pull request:
Removed EXPOSE 443 in yaml that was unneccessary.
Used .NET 5 links

Update descriptions as .NET Core now is .NET(also the visual studio workload)

## Summary

Describe your changes here.

Fixes #25098

macel94 committed:
Update getting-started.md to use .NET 5

Removed EXPOSE 443 in yaml that was unneccessary.
Used .NET 5 links

Update descriptions as .NET Core now is .NET(also the visual studio workload)

dnfadmin commented:
[![CLA assistant check](https://cla.dotnetfoundation.org/pull/badge/not_signed)](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=25117) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=25117) before we can accept your contribution.<br/><br/>:x: MACEL94 [sign now](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=25117)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/docs?pullRequest=25117) it.</sub>

MACEL94 commented:
#25098

amolenk commented on docs/architecture/dapr-for-net-developers/getting-started.md:
```diff
@@ -371,34 +371,33 @@ In the final part of this example, you'll add container support and run the solu
+    FROM mcr.microsoft.com/dotnet/aspnet:5.0 AS base
     WORKDIR /app
     EXPOSE 80
-    EXPOSE 443
```
Leave the 443 port in for the front-end. If you follow the instructions closely, you only disable HTTPS for the back-end project. The Docker file that VS generates for the front-end will contain both 80/443 ports.

amolenk commented on docs/architecture/dapr-for-net-developers/getting-started.md:
```diff
@@ -371,34 +371,33 @@ In the final part of this example, you'll add container support and run the solu
+    
+    1. Pulls the `mcr.microsoft.com/dotnet/aspnet:5.0` image and names it `base`.
     1. Sets the working directory to */app*.
-    1. Exposes port `80` and `443`.
```
See previous comment; include 443.

amolenk reviewed (changes requested):
Thanks for the PR! 

I've only got comments on the 443 port in the front-end.

@IEvangelist Can you review as well? 🙏

macel94 committed:
Update getting-started.md

Fix as requested in the comments.
443 port in frontend left exposed to follow the tutorial instructions.

IEvangelist reviewed (approved):
Looks good to me, we'll :shipit: - thank you 🙏

IEvangelist committed:
Update getting-started.md
